Makefile

```diff
@@ -37,12 +37,15 @@ pypi_upload: install
 	$(call header,"Uploading new version to PyPi")
 	@. $(INSTALL_DIR)/bin/activate; python setup.py sdist upload -r pypi
 
-docs: install
-	$(call header,"Building docs")
+clean_docs: install
```

Comment:
I'm ok with removing it. I guess if we ever needed a complete reset we can just run the commands manually.